suite/tests/api/ir_riscv64.expect

```diff
@@ -311,15 +312,15 @@ sd     a0 -> (a1)[8byte]
 sb     a0 -> (a1)[1byte]
 sh     a0 -> (a1)[2byte]
 addi   a1 0 -> a0
-jalr   a0 0x0 -> zero
```

Comment:
Isn't it nicer if they are hex?

Reply:
Ah, I simply followed the objdump style:

```
$ objdump -b binary -m riscv -D testfile

testfile:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	064500e7          	jalr	100(a0)
```